reactos/drivers/storage/storahci/storahci.c

```diff
@@ -230,64 +295,67 @@ AhciInterruptHandler (
  *
  * The Storport driver calls the HwStorInterrupt routine after the HBA generates an interrupt request.
  *
- * @param adapterExtension
+ * @param AdapterExtension
  *
  * @return
  * return TRUE Indicates that an interrupt was pending on adapter.
  * return FALSE Indicates the interrupt was not ours.
  */
 BOOLEAN
 AhciHwInterrupt(
-    __in PVOID AdapterExtension
+    __in PAHCI_ADAPTER_EXTENSION AdapterExtension

```

Comment:
I'm surprised the compiler doesn't complain about this. Aren't storport's function pointers strongly typed?